as/plugins/org.jboss.ide.eclipse.as.classpath.core/src/org/jboss/ide/eclipse/as/classpath/core/runtime/modules/manifest/AbstractModuleSlotUtil.java

```diff
@@ -174,7 +176,9 @@ public void ensureInCache(IFile f) {
 			try {
 				cacheRelevantFiles(p);
 			} catch(CoreException ce) {
-				ClasspathCorePlugin.getDefault().getLog().log(ce.getStatus());
+				IStatus o = ce.getStatus();
+				IStatus stat2 = new Status(o.getSeverity(), o.getPlugin(), o.getMessage(), ce);
+				ClasspathCorePlugin.getDefault().getLog().log(stat2);
```

Comment:
When looking through the aeri log, it didn't have the original coreexception's trace.  The trace started right here and only had the original CE's message. ClasspathCorePlugin doesn't have the pluginLog() method from foundation, so yeah, a bit more fuss.